theme/aplus/static/default.css

```diff
@@ -97,467 +108,528 @@ dt:hover > a.headerlink,
+  vertical-align: middle;
 }
 
 div.body div.math p {
-    text-align: center;
+  text-align: center;
 }
 
 span.eqno {
-    float: right;
+  float: right;
 }
 
 span.eqno a.headerlink {
-    position: relative;
-    left: 0px;
-    z-index: 1;
+  position: relative;
+  left: 0px;
+  z-index: 1;
 }
 
 div.math:hover a.headerlink {
-    visibility: visible;
+  visibility: visible;
 }
 
 /* -- printout stylesheet --------------------------------------------------- */
 
 @media print {
-    div.document,
-    div.documentwrapper,
-    div.bodywrapper {
-        margin: 0 !important;
-        width: 100%;
-    }
+  div.document,
+  div.documentwrapper,
+  div.bodywrapper {
+    margin: 0 !important;
+    width: 100%;
+  }
 
-    div.sphinxsidebar,
-    div.related,
-    div.footer,
-    #top-link {
-        display: none;
-    }
+  div.sphinxsidebar,
+  div.related,
+  div.footer,
+  #top-link {
+    display: none;
+  }
 }
 
 /* -- point of interest --------------------------------------------------- */
 
 .poi {
-    margin: 20px 0;
-    padding: 0;
-    width:inherit;
+  margin: 20px 0;
+  padding: 0;
+  width: inherit;
 }
 
 .no-poi-box {
-    margin: 0;
-    padding: 0;
-    width: inherit;
+  margin: 0;
+  padding: 0;
+  width: inherit;
 }
 
 .poi-container {
-    width:100%;
-    padding:0;
-    margin: 5px 0;
+  width: 100%;
+  padding: 0;
+  margin: 5px 0;
 }
 
 .poi-content {
-    border: 1px solid #ccc;
-    padding:15px 20px;
-    border-radius:4px;
-}   
+  border: 1px solid #ccc;
+  padding: 15px 20px;
+  border-radius: 4px;
+}
 
 .poi-title {
-    width: 80%;
-    float:left;
-    height:20px;
-    font-weight: bold;
-    font-size:1.2em;
-    padding-left:0;
-} 
+  width: 80%;
+  float: left;
+  height: 20px;
+  font-weight: bold;
+  font-size: 1.2em;
+  padding-left: 0;
+}
 
 .poi-links {
-    width:20%;
-    text-align:right;
-    float:left;
-    margin: 5px 0 0 0;
-    height:20px;
+  width: 20%;
+  text-align: right;
+  float: left;
+  margin: 5px 0 0 0;
+  height: 20px;
 }
 
 .poi-icon {
-    margin: 0 10px;
-    vertical-align: baseline;
-    height:20px;
+  margin: 0 10px;
+  vertical-align: baseline;
+  height: 20px;
 }
 
 .borderless .poi-content {
-    border: 0;
+  border: 0;
 }
 
 .poi-container.poi-content {
-    border: 1px solid #e7f2fa;
-    border-top: 2px solid #3c763d;
-    border-bottom: 2px solid #3c763d;
-    background: #ebf5e7;
+  border: 1px solid #e7f2fa;
+  border-top: 2px solid #3c763d;
+  border-bottom: 2px solid #3c763d;
+  background: #ebf5e7;
+}
+
+/* The current version of A+ use Bootstrap 3. The following classes will emulate 
+some of the changes that are implemented in Bootstrap 4  */
+body {
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
+    "Helvetica Neue", Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji",
+    "Segoe UI Symbol";
+  /* font-size:  1rem; */
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 1.6;
+  color: #212529;
```

Comment:
So, this is a bit more "black" and tinted to blue than the one in bootstrap 3?

Reply:
yes, the color increase the contrast from 12 to 15. Nothing expecial here but it is an small improvement :)

Reply:
And thus should be done in A+, not here...